cfy_manager/components/composer/composer.py

```diff
@@ -68,6 +71,74 @@ def _run_db_migrate(self):
             ],
         )
 
+    def _handle_ca_certificate(self, installing):
+        base_cert_config = {
+            'logger': self.logger,
+            'ca_destination': DB_CA_PATH,
+            'owner': COMPOSER_USER,
+            'group': COMPOSER_GROUP
+        }
+
+        install_cert_config = {
+                'component_name': POSTGRESQL_CLIENT,
```

Comment:
Not major, but this indentation could probably stand to lose some weight.

Reply:
Meaning, changing to: 
install_cert_config = {'component_name': POSTGRESQL_CLIENT, 
                                  'update_config': False}
?